CHANGELOG.md

```diff
@@ -16,6 +16,19 @@ Allowed headings:
 
 ## [Unreleased]
 
+### Changed
+
+* Updated the following:
+	- targetSdk/compileSdk -> 31
```

Comment:
It would be nice to mention where we are going from as well as where to e.g. from junit 4.12 -> 4.13.2 (made up numbers)